### PR TITLE
Landing page migration, TOC tweaks

### PR DESCRIPTION
**Pull request by DarrinEide**

### CODE REVIEW

**Comment by DarrinEide** [comment]
See https://review.docs.microsoft.com/en-us/academic-services/?branch=DarrinEide%2FMigrateLandingPages to review

File: academic-services/knowledge-exploration-service/index.yml [hunk 1]
```diff
@@ -1,39 +1,65 @@
...
+      - linkListType: overview
+        links:
+          - text: What is MAKES?
+            url: introduction.md
+      - linkListType: concept
+        links:
+          - text: How MAKES understands natural language queries
+            url: concept-queries.md
+
+  - title: Get started
```
Comment (alch-msft) [inline academic-services/knowledge-exploration-service/index.yml:26]: > Get started [](start = 10, length = 12)

Get Started section already exist on the main navigation rail. How about moving the two tutorials under "Get started" section to "How-To" and have this page be the entry point of "Get-Started"?

File: academic-services/knowledge-exploration-service/index.yml [hunk 2]
```diff
@@ -1,39 +1,65 @@
...
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Sign up for MAKES
+            url: get-started-setup-provisioning.md
+      - linkListType: how-to-guide
+        links:
+          - text: Deploy new MAKES instance
```
Comment (alch-msft) [inline academic-services/knowledge-exploration-service/index.yml:34]: >Deploy new MAKES instance [](start = 18, length = 25)

Let's modify "Create an API instance" to "Deploy new MAKES instance" on the main navigation rail to be consistent.

File: academic-services/knowledge-exploration-service/index.yml (hunk 1 above)
Comment (DarrinEide) [reply academic-services/knowledge-exploration-service/index.yml:26]: Hmm... I think having a "getting started" section is important in the navigation rail, as it's something folks are accustomed to looking for.

How about instead we simply rename this section on the landing page to "Top tasks"?

---
In reply to: [463834513](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/169#discussion_r463834513) [](ancestors = 463834513)

**Pull request merged**